<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QWidget" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;123&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item>
    <widget class="QLabel" name="label_timer">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Roboto&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>подтвердить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start_game">
       <property name="text">
        <string>играем!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Anime_btn">
       <property name="text">
        <string>Anime</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Anime_btn</sender>
   <signal>clicked()</signal>
   <receiver>mainForm</receiver>
   <slot>AnimeButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AnimeButton_clicked()</slot>
 </slots>
</ui>
